pypicloud/static/partial/package.html

```diff
@@ -61,6 +62,9 @@ <h2>{{ package_name }}</h2>
                 Delete
               </button>
             </td>
+            <td>
+              {{ package.metadata.uploader !== null ? package.metadata.uploader : "&lt;unknown&gt;" }}
```

Comment:
fix: You'll want to use `!=` here so that it shows `<unknown>` when the package doesn't have an uploader in the metadata

Reply:
Done :)
I tested it before using `!=`, and it worked, probably didn't re-check after changing to `!==` as I saw it's more correct to use in JS.
Restored to `!=~

Reply:
Yeah in JS `!==` is generally preferred as it does what `!=` does in most other languages, but most people are willing to make an exception for `!= null` because it's a useful shorthand for "is not null or undefined".

Reply:
Good to know :)
Any more changes you would like me to add?